hotshot-state-prover/src/service.rs

```diff
@@ -478,11 +480,15 @@ mod test {
         let signer = Wallet::from(anvil.keys()[0].clone());
         let l1_wallet = Arc::new(L1Wallet::new(provider.clone(), signer));
 
-        Command::new("just")
+        let status = Command::new("just")
```

Comment:
Not an issue with this PR later but I'm not sure it's a great idea to call `just` from our rust tests. There doesn't seem to be much benefit from using `just` here it but we've created another dependency for a tool that isn't so commonplace and now needs to be installed to run `cargo test`. Any downside of using a bash script? What do others think about this?